app/models/student.rb

```diff
@@ -7,15 +13,16 @@ class Student < ApplicationRecord
   has_many :assessments
 
   has_many :affiliates
+  has_many :student_comments
 
   has_many :taggings
   has_many :tags, through: :taggings
 
   validates :first_name, presence: true
-  # validates :last_name, presence: true
+  validates :last_name, presence: true
   # validates :dob, presence: true
-  # validates :gender, presence: true
-  # validates :smartt_id, presence: true
+  validates :gender, presence: true
+  validates :smartt_id, presence: true
```

Comment:
It should allow for a blank `smartt_id` for the same reason as a tutor: they are only given when entering the data into smartt and not necessarily known at creation time in our portal.